### DISCUSSION TITLE
Possible to support tokio::task::LocalKey?

### DISCUSSION THREAD

**Discussion by lucasyvas** [napi-rs/napi-rs#1494] [Ideas]
I had an idea for a native module that I would like to support some type of "unit of work" type context. This would potentially require setting a LocalKey on the node side, but having the value be consumable on the Rust side.

This is handy for having context-aware log messages, database transactions, etc. This might be practically difficult to achieve, but I'm not sure!

**Comment by Brooooooklyn** [comment]
https://napi.rs/docs/concepts/external

**Reply by lucasyvas** [reply]
Wow, never saw this! Will take a look - thanks.
